dyson/chain.json

```diff
@@ -0,0 +1,104 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "dyson",
+  "website": "https://dysonprotocol.com/",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "Dyson Protocol",
+  "chain_id": "dyson-mainnet-01",
+  "bech32_prefix": "dys",
+  "node_home": "$HOME/.dyson",
+  "daemon_name": "dysond",
+  "slip44": 118,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "dys",
+        "low_gas_price": 0.001,
+        "average_gas_price": 0.002,
+        "high_gas_price": 0.01
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "dys"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://gitlab.com/dysonproject/dyson",
+    "recommended_version": "v0.0.0-rc1",
+    "compatible_versions": [
+      "v0.0.0-rc1"
+    ],
+    "genesis": {
+      "genesis_url": "https://gitlab.com/dysonproject/dyson-deploy/-/raw/develop/genesis.json"
+    }
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "2c4f8e193fd10ab3a2bc919b484fd1c78eceffb3",
+        "address": "85.159.211.61:26656",
+        "provider": ""
+      },
+      {
+        "id": "1f3bd1f48c7c277e7be83895cdbb46f8333adb61",
+        "address": "178.79.183.171:26656",
+        "provider": ""
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "2c4f8e193fd10ab3a2bc919b484fd1c78eceffb3",
+        "address": "85.159.211.61:26656",
+        "provider": ""
+      },
+      {
+        "id": "2c4f8e193fd10ab3a2bc919b484fd1c78eceffb3",
+        "address": "85.159.211.61:26656",
+        "provider": ""
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://dys-tm.dyson.lol:443",
+        "provider": "lol"
+      },
+      {
+        "address": "https://dys-tm.dysonprotocol.com:443",
+        "provider": "dysonprotocol"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://dys-api.dyson.lol:443",
+        "provider": "lol"
+      },
+      {
+        "address": "https://dys-api.dysonprotocol.com:443",
+        "provider": "dysonprotocol"
+      }
+    ],
+    "grpc": [
+      "https://dys-grpc.dyson.lol:443"
```

Comment:
I think this needs to be in an endpoint or api object

Reply:
It's not the grpc endpoint?
```
$ grpcurl dys-grpc.dyson.lol:443 list
cosmos.auth.v1beta1.Query
cosmos.authz.v1beta1.Query
cosmos.bank.v1beta1.Query
cosmos.base.reflection.v1beta1.ReflectionService
cosmos.base.reflection.v2alpha1.ReflectionService
cosmos.base.tendermint.v1beta1.Service
cosmos.distribution.v1beta1.Query
cosmos.evidence.v1beta1.Query
cosmos.feegrant.v1beta1.Query
cosmos.gov.v1.Query
cosmos.gov.v1beta1.Query
cosmos.group.v1.Query
cosmos.mint.v1beta1.Query
cosmos.params.v1beta1.Query
cosmos.slashing.v1beta1.Query
cosmos.staking.v1beta1.Query
cosmos.tx.v1beta1.Service
cosmos.upgrade.v1beta1.Query
dyson.Query
grpc.reflection.v1alpha.ServerReflection
ibc.applications.interchain_accounts.controller.v1.Query
ibc.applications.interchain_accounts.host.v1.Query
ibc.applications.transfer.v1.Query
ibc.core.channel.v1.Query
ibc.core.client.v1.Query
ibc.core.connection.v1.Query
names.Query
```

Reply:
I used this as a reference: https://github.com/cosmos/chain-registry/blob/master/cosmoshub/chain.json#L366
```
$ grpcurl cosmoshub-grpc.lavenderfive.com:443 list
cosmos.auth.v1beta1.Query
cosmos.authz.v1beta1.Query
cosmos.bank.v1beta1.Query
cosmos.base.reflection.v1beta1.ReflectionService
cosmos.base.reflection.v2alpha1.ReflectionService
cosmos.base.tendermint.v1beta1.Service
cosmos.distribution.v1beta1.Query
cosmos.evidence.v1beta1.Query
cosmos.feegrant.v1beta1.Query
cosmos.gov.v1beta1.Query
cosmos.mint.v1beta1.Query
cosmos.params.v1beta1.Query
cosmos.slashing.v1beta1.Query
cosmos.staking.v1beta1.Query
cosmos.tx.v1beta1.Service
cosmos.upgrade.v1beta1.Query
grpc.reflection.v1alpha.ServerReflection
ibc.applications.interchain_accounts.controller.v1.Query
ibc.applications.interchain_accounts.host.v1.Query
ibc.applications.transfer.v1.Query
ibc.core.channel.v1.Query
ibc.core.client.v1.Query
ibc.core.connection.v1.Query
router.v1.Query
tendermint.liquidity.v1beta1.Query
```

Reply:
Nevermind, I get it now, it was the schema failing

Reply:
Ok rebased from master again, had a small snafu